qdrant-landing/content/documentation/guides/installation.md

```diff
@@ -12,7 +12,11 @@ The following sections describe the requirements for deploying Qdrant.
 
 ### CPU and memory
 
-The CPU and RAM that you need depends on:
+Your CPU must meet the following requirements:
+
+- x86_64/amd64 or AArch64/arm64, 32-bit systems are not supported
```

Comment:
Since we are using bulletpoints, maybe it could be more understandable like this:
```suggestion
CPU architectures:
- 64-bit system 
  - x86_64/amd64 
  - AArch64/arm64
- 32-bit system
  - Not supported
```